src/test/ui/consts/const-eval/maybe_uninit.rs

```diff
@@ -0,0 +1,36 @@
+// check-pass
+
+#![feature(const_mut_refs)]
+#![cfg_attr(not(bootstrap), feature(const_maybe_uninit_as_mut_ptr))]
+#![cfg_attr(bootstrap, feature(const_maybe_uninit_as_ptr))]
+#![allow(dead_code)]
+
+use std::mem::MaybeUninit;
+use std::ptr;
+
+#[derive(Debug, PartialEq)]
```

Comment:
Actually just noticed that this line isn't necessary. I considered a run-pass where the value was asserted but didn't feel it necessary.

Reply:
Why is this a ui test? It could be a unit test (and those are typically preferred, I think, since they are cheaper).

Reply:
Honestly no idea; that's just where I've put tests in the pass. Where should I put this instead?

Reply:
https://github.com/rust-lang/rust/blob/master/library/core/tests/mem.rs looks like a good place.

Reply:
Done.